python/ray/autoscaler/resource_demand_scheduler.py

```diff
@@ -62,6 +62,8 @@ def get_nodes_to_launch(self, nodes: List[NodeID],
             logger.info("No resource demands")
             return []
 
+        logger.info("Available resources: {}".format(usage_by_ip))
```

Comment:
Isn't this kind of verbose?

Reply:
i think this is approximately the same size as `node_resources` (since each node has a custom resource). I can remove it though.